integration-libs/cpq-quote/cpq-quote-discount/components/cpq-quote/cpq-quote-offer.component.scss

```diff
@@ -0,0 +1,8 @@
+.offer {
```

Comment:
This line should be removed after `cpq-quote-offer.component.scss` file has been moved to `styles` folder.
By the way all class should start with `cx-` prefix